global.json

```diff
@@ -1,6 +1,6 @@
 {
 	"sdk": {
-		"version": "3.1.300",
+		"version": "6",
```

Comment:
If you set a specific version here `actions/setup-dotnet` will read this file, and you can omit `dotnet-version` setting. `6.0.400` is the latest version.

Reply:
Fixed